fcrepo-import-export-driver/src/main/java/org/fcrepo/importexport/ArgParser.java

```diff
@@ -123,13 +202,42 @@ Config parseConfiguration(final String[] args) {
             config.setRdfExtension(cmd.getOptionValue('x', DEFAULT_RDF_EXT));
             config.setRdfLanguage(cmd.getOptionValue('l', DEFAULT_RDF_LANG));
 
-        } catch (ParseException e) {
+        } catch (final ParseException e) {
             printHelp("Error parsing args: " + e.getMessage());
         }
         return config;
     }
 
     /**
+     * This method writes the configuration file to disk
+     *
+     * @param args to be persisted
+     */
+    private void saveConfig(final String[] args) {
+        final File configFile = new File(System.getProperty("java.io.tmpdir"), CONFIG_FILE_NAME);
```

Comment:
I don't understand why anything is getting put in the tmp directory here, but if you're going to do that, use [this](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#createTempFile-java.lang.String-java.lang.String-java.nio.file.attribute.FileAttribute...-).